synapse_rock/rockcraft.yaml

```diff
@@ -106,6 +106,10 @@ parts:
             cd synapse_stats_exporter
             pip3 install --prefix="/install" --no-warn-script-location -U .
             cd ..
+            git clone https://github.com/matrix-org/synapse-s3-storage-provider
```

Comment:
Should we clone a specific tag?
Like v1.4.0 is the latest one.

Reply:
will do so